### ISSUE TITLE
calendar button not work on Search when use BETWEEN condition popup window (datetime field type)

### ISSUE THREAD

**Issue by AndreyPopovNew** [phpmyadmin/phpmyadmin#18806]
### Describe the bug

when use Search in table and choose BETWEEN condition then in popup windows calendar button not open calendar window (datetime type field)

### To Reproduce

1. Go to any table
2. Click on Search
3. Scroll down to any date-time format based field
4. choose BETWEEN condition
5. in opened popup windows try tap calendar button - nothing happen

### Expected behavior

must open calendar window to choose date and time

### Screenshots

![phpmyadmin_calendar](https://github.com/phpmyadmin/phpmyadmin/assets/8410116/cd8013f2-0c57-47f1-89ef-5d7f49853dba)

### Server configuration

- Operating system:
- Web server: Apache libmysql - mysqlnd 7.4.33
- Database version: 8.0.34-26 - Percona Server (GPL), Release 26, Revision 0fe62c85
- PHP version: 7.4.33
- phpMyAdmin version: 5.2.1

### Client configuration

- Browser: Chrome, Edge, Opera
- Operating system: Windows 10/11

**Comment by kaneprajakta** [comment]
Hello Andrey , This is Praju , trying to reproduce the bug , Can you help me getting the exact version
Web server: Apache libmysql - mysqlnd 7.4.33
Database version: 8.0.34-26 - Percona Server (GPL), Release 26, Revision 0fe62c85
PHP version: 7.4.33
phpMyAdmin version: 5.2.1
on my widows 11 system , I am using wamp server on windows 11

**Comment by AndreyPopovNew** [comment]
> Hello Andrey , This is Praju , trying to reproduce the bug , Can you help me getting the exact version Web server: Apache libmysql - mysqlnd 7.4.33 Database version: 8.0.34-26 - Percona Server (GPL), Release 26, Revision 0fe62c85 PHP version: 7.4.33 phpMyAdmin version: 5.2.1 on my widows 11 system , I am using wamp server on windows 11

shared hosting ukraine.com.ua

![image_2024-01-03_163209985](https://github.com/phpmyadmin/phpmyadmin/assets/8410116/34e8231a-89b7-49cf-858c-00c37f4b35ee)


web server is Apache

Windows only for client connection

**Comment by kaneprajakta** [comment]
I tried reproducing on my system . The config is not exactly same but almost same
![Screenshot 2024-01-03 233505](https://github.com/phpmyadmin/phpmyadmin/assets/47685334/c8c872cb-e7a4-4489-888b-7d0cd84b4d43)
I can see the date drop down in chrome.

**Comment by AndreyPopovNew** [comment]
> I tried reproducing on my system . The config is not exactly same but almost same  can see the date drop down in chrome.


you try field type "date"
I try field type "datetime"

![image_2024-01-03_201615907](https://github.com/phpmyadmin/phpmyadmin/assets/8410116/36b3c7c7-e1ad-4f6a-a12f-1bd5531df2d2)

but on BETWEEN POPUP calendar not drop down

**Comment by williamdes** [comment]
Hi @faissaloux 
would you help with this issue that seems to be a closing and opening modal issue ?
target branch `QA_5_2`

**Comment by faissaloux** [comment]
In my case: When click on the calendar icon the first time it works, if closed and want to open second time it does not. And when open the other field calendar and back to the first it always works in the first time.

@AndreyPopovNew Can you confirm is this your case or it does not open at all, because I couldn't reproduce it here.

**Comment by AndreyPopovNew** [comment]
@faissaloux 
in my database - date_added  (type datetime)
when choose BETWEEN - calendar never opened on click in popup windows

**Comment by faissaloux** [comment]
I couldn't reproduce this on my side.

@williamdes did you?

**Comment by AndreyPopovNew** [comment]
on https://demo.phpmyadmin.net/QA_5_2/
all work. calendar appear in popup window always.

**Comment by faissaloux** [comment]
Yeah I think it's fixed, you just have to update your phpmyadmin.

**Comment by williamdes** [comment]
Awesome, 5.2.2 should be out very shortly
